packages/web/src/types.ts

```diff
@@ -444,3 +444,17 @@ export interface HttpNetworkConfig {
 }
 
 export { WAIT_ROUNDS };
+
+export interface ConfirmedTxInfo {
+	"confirmed-round": number;
+	"asset-index": number;
+	"application-index": number;
+	"global-state-delta"?: algosdk.modelsv2.EvalDeltaKeyValue;
+	"local-state-delta"?: algosdk.modelsv2.AccountStateDelta;
+	"inner-txns"?: ConfirmedTxInfo;
+	txn: algosdk.EncodedSignedTransaction;
+}
+
+export interface TxnReceipt extends ConfirmedTxInfo {
+	txID: string;
+}
```

Comment:
For js i think the indentation is 2 spaces. Most of the code in other files have that. Could you please check/verify?

Reply:
It is consistent with other files, I checked